packages/salesforcedx-vscode-core/src/util/authInfo.ts

```diff
@@ -11,10 +11,14 @@ import { channelService } from '../channels';
 import { nls } from '../messages';
 import { notificationService } from '../notifications';
 import { telemetryService } from '../telemetry';
-import { ConfigSource, ConfigUtil } from './index';
+import {
+  ConfigSource,
+  ConfigUtil,
+} from './index';
+
+export const defaultUserNameKey = 'defaultusername';
+export const defaultDevHubUserNameKey = 'defaultdevhubusername';
```

Comment:
Can we add `defaultUserNameKey` and `defaultDevHubUserNameKey` to the [constants](https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/packages/salesforcedx-vscode-core/src/constants.ts) file since it's being used it more than one place now?